content/influxdb/v2.0/reference/flux/stdlib/experimental/quantile.md

```diff
@@ -0,0 +1,109 @@
+---
+title: experimental.quantile() function
+description: >
+  The `experimental.quantile()` function outputs non-null records with values in
+  the `_value` column that fall within the specified quantile or the non-null
+  record with the value in the `_value` column that represents the specified quantile.
+menu:
+  influxdb_2_0_ref:
+    name: experimental.quantile
+    parent: Experimental
+weight: 302
+related:
+  - /influxdb/v2.0/query-data/flux/percentile-quantile/
+  - /influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/quantile/
+  - /{{< latest "influxdb" "v1" >}}/query_language/functions/#percentile, InfluxQL – PERCENTILE()
+---
+
+The `experimental.quantile()` function outputs non-null records with values in
+the `_value` column that fall within the specified quantile or the non-null
+record with the value in the `_value` column that represents the specified quantile.
```

Comment:
```suggestion
The `experimental.quantile()` function outputs non-null records with values in
the `_value` column that fall within the specified quantile or represent the specified quantile.
```

Reply:
Not clear on the difference between fall within and represent here, if we can clarify